source/beamline/AMEnumeratedControl.h

```diff
@@ -11,6 +11,9 @@ class AMEnumeratedControl : public AMPseudoMotorControl
     Q_OBJECT
 
 public:
+	/// Enumeration of some default option indices.
+	enum DefaultOptions { Unknown = -1, Invalid = -2 };
```

Comment:
If there is a plan for them then I don't see a reason to remove them.  Only if they aren't needed or desired.